nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/HealthCheckService.java

```diff
@@ -208,13 +209,16 @@ public void bootstrapHealthCheck() {
     /**
      * Starts all notifiers that have been enable with the healthcheck service.
      */
-    public void bootstrapNotifierList() {
+    public synchronized void bootstrapNotifierList() {
```

Comment:
I think this should be `synchronized` to be consistent in case of concurrent service reboots (e.g. possible by concurrent admin commands).